sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/ThroughputControlStore.java

```diff
@@ -159,9 +147,37 @@ public void enableThroughputControlGroup(ThroughputControlGroupInternal group) {
                 // We will handle the first scenario by creating a new container controller,
                 // while fall back to original request Mono for the second scenario.
                 return this.updateControllerAndRetry(collectionNameLink, request, originalRequestMono);
+            })
+            .onErrorResume(throwable -> {
+               if (throwable instanceof ThroughputControlInitializationException) {
+                      if (this.shouldContinueRequestOnInitError(request, collectionNameLink, throwable)) {
```

Comment:
Since we want to continue here, based on if condition, I am wondering if we should use `onErrorResume()` or `onErrorContinue()` ?

Example -> https://devdojo.com/ketonemaniac/reactor-onerrorcontinue-vs-onerrorresume

Reply:
yea, it makes sense. But I think using onErrorResume maybe more cleaner here? as based on different conditions, we are behaving kind differently. Will discuss more offline.